### PR TITLE
Increment version for spring-modules release

### PR DESCRIPTION
**Pull request by chenrujun**
Change version for spring modules.

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @yiliuTo , please help to review this PR. 🙏

File: sdk/spring/azure-spring-boot-starter-data-gremlin/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 3.0.0-beta.1 (Unreleased)
+## 3.0.0-beta.1 (2021-01-20)
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-boot-starter-data-gremlin/CHANGELOG.md:3]: This file needn't to be modified.

File: sdk/spring/azure-spring-boot-starter/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 3.2.0-beta.1 (Unreleased)
+## 3.1.0 (2021-01-20)
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-boot-starter/CHANGELOG.md:3]: We need to add all spring boot related changelogs here.

File: sdk/spring/azure-spring-boot/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 3.2.0-beta.1 (Unreleased)
+## 3.1.0 (2021-01-20)
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-boot/CHANGELOG.md:3]: Same as the azure-spring-boot-starter

File: sdk/spring/azure-spring-boot-starter-storage/CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Release History
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-boot-starter-storage/CHANGELOG.md:1]: Readme of storage starter is missing, and it's due to the wrong label of the storage starter inside the readme, please help to update it together.

File: sdk/spring/azure-spring-boot-starter-data-gremlin/CHANGELOG.md (hunk 1 above)
Comment (chenrujun) [reply sdk/spring/azure-spring-boot-starter-data-gremlin/CHANGELOG.md]: Updated.

File: sdk/spring/azure-spring-boot-starter/CHANGELOG.md (hunk 2 above)
Comment (chenrujun) [reply sdk/spring/azure-spring-boot-starter/CHANGELOG.md:3]: Updated.

File: sdk/spring/azure-spring-boot/CHANGELOG.md (hunk 3 above)
Comment (chenrujun) [reply sdk/spring/azure-spring-boot/CHANGELOG.md:3]: Updated.

File: sdk/spring/azure-spring-boot-starter-storage/CHANGELOG.md (hunk 4 above)
Comment (chenrujun) [reply sdk/spring/azure-spring-boot-starter-storage/CHANGELOG.md:1]: Updated.

**Review by yiliuTo — APPROVED** [review]
LGTM

**Pull request merged**